docs/certificate-guides/exchange-inbox.md

```diff
@@ -47,15 +51,37 @@ Once you withdraw the Mwh, they  will be transferred to your [Blockchain Inbox](
 
 ## Transfer
 
-Mwh from certificates can be transferred to another Exchange Deposit address if you would like to perform a transfer or a sale off of the exchange interface. To transfer Mwh from a certificate to another Exchange inbox account, select the certificate that you would like to transfer and navigate to the Transfer tab. 
+Mwh from certificates can be transferred to another Exchange Deposit address (The public address associated with another Exchange Deposit account) if you would like to perform a transfer or a sale outside of the Exchange interface. To transfer Mwh from a certificate to another Exchange inbox account, select the certificate that you would like to transfer and navigate to the Transfer tab. 
 
 ![certificate-exchangeinbox-transfer](../images/certificates/certificate-exchangeinbox-transfer.png)
 
 Note that you can edit the number of Mwh you would like to transfer by clicking the pencil icon next to the Mwh. By default, the number of Mwh to transfer is the total number of Mwh available from the certificate. 
 
 Enter the exchange deposit address and click “Transfer Certificate”. 
 
-## Export
+## Claim
+
+Mwh from certificates are claimed when you would like to use them for sustainability reporting, and keep them permanently in the Exchange wallet. Once hours are claimed, they will no longer be visible in your Exchange Inbox. They are permanently locked in place in the Exchange wallet and cannot be revoked by the issuer, transferred or withdrawn from the Exchange wallet. 
```

Comment:
Also important note that:
If the user doesn't have blockchain account linked - Certificates claimed from the Exchange inbox will be automatically claimed to the Platform Blockchain address, but with the user's organization info in the Claim Data